src/AWS.Deploy.Orchestration/Orchestrator.cs

```diff
@@ -244,6 +244,13 @@ public async Task ApplyAllReplacementTokens(Recommendation recommendation, strin
                 recommendation.AddReplacementToken(Constants.RecipeIdentifier.REPLACE_TOKEN_DEFAULT_CONTAINER_PORT, defaultPort);
                 recommendation.DeploymentBundle.DockerfileHttpPort = defaultPort;
             }
+            if (recommendation.ReplacementTokens.ContainsKey(Constants.RecipeIdentifier.REPLACE_TOKEN_DEFAULT_ENVIRONMENT_ARCHITECTURE))
+            {
+                if (_dockerEngine == null)
```

Comment:
Why is docker required for architecture? What if we are building a deployment package for Beanstalk?

Reply:
Docker is not required. I removed the check.